odxtools/__init__.py

```diff
@@ -62,11 +62,11 @@
 - _`[ISO22901]` The ISO 22901 Standard: https://www.iso.org/standard/41207.html
 
 """
-from .load_file import load_file
-from .load_odx_d_file import load_odx_d_file
-from .load_pdx_file import load_pdx_file
-from .version import __version__
-from .write_pdx_file import write_pdx_file
+from .load_file import load_file as load_file
+from .load_odx_d_file import load_odx_d_file as load_odx_d_file
+from .load_pdx_file import load_pdx_file as load_pdx_file
+from .version import __version__ as __version__
+from .write_pdx_file import write_pdx_file as write_pdx_file
```

Comment:
why?

Reply:
because importing without `as Foo` leads to `mypy --strict` to produce
```
odxtools/write_pdx_file.py:141: error: Module "odxtools" does not explicitly export attribute "__version__"  [attr-defined]
```